composer.json

```diff
@@ -32,8 +31,7 @@
         "symfony/security-bundle": "To be able to use the publish workflow system (^2.1)",
         "symfony-cmf/routing": "To be able to use the CMF twig extension functions cmf_prev_linkable/cmf_next_linkable (^1.2)",
         "symfony-cmf/routing-bundle": "To be able to enable the publish_workflow_listener (^1.2)",
-        "sonata-project/admin-bundle": "To be able to enable the publish_workflow_listener extension (^2.2)",
-        "sonata-project/translation-bundle": "To be able to edit translatable documents (^1.0)"
+        "symfony-cmf/sonata-admin-integration": "To be able to enable the publish_workflow_listener extension (^2.2)"
```

Comment:
from the view of the core-bundle it is "just" the usage of some extension, cause the core-bundle has no admins :-) but i can take the one i have in the others.